lib/core/src/lib.rs

```diff
@@ -27,10 +27,18 @@ pub mod system {
 
 use display::world::World;
 use wasm_bindgen::prelude::*;
+use basegl_system_web::keyboard_engine::KeyboardEngine;
 
 #[wasm_bindgen(start)]
 pub fn start() {
     let world = World::new();
     world.add_workspace("canvas");
     world.start();
+
+    let mut ke = KeyboardEngine::from_tag_name("input", true);
```

Comment:
Let's attach the KeyboardEngine to the whole screen, not only the input box. 
Plus, let's use more descriptive names. Arguably, we find `keyboard_engine` better name than `ke`.